# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/mask/mask-perspective.png
# Commit: b641b7f90108
# Full Hash: b641b7f90108a2e6b57186139b450d9ccb80c96c
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2021-04-10 03:36:58
# Regressor Bug: 1682195
# File Overlap Count: 1
# Description:
#   Bug 1682195 - Use scissoring to restrict cs_clip_image rather than discard. r=gw
#   
#   cs_clip_image renders the entire target-space sub-rect, and then uses discard against
#   the local-space bounds to ensure the primitive gets trimmed down to its actual footprint
#   within the larger target-space sub-rect. This can be fairly wasteful and slow.
# ==============================================================================

diff -r 7534ee1e1078 -r b641b7f90108 gfx/wr/wrench/reftests/mask/mask-perspective.png
Binary file gfx/wr/wrench/reftests/mask/mask-perspective.png has changed